lv2/CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ set (LV2PLUGIN_PRJ_NAME "${PROJECT_NAME}_lv2")
 # Set the build directory as <build_dir>/lv2/<plugin_name>.lv2/
 set (PROJECT_BINARY_DIR "${PROJECT_BINARY_DIR}/${PROJECT_NAME}.lv2")
 
+# C99 or higher is needed
+set(CMAKE_C_STANDARD 99)
```

Comment:
Move this to `SfizzConfig.cmake` ?

Reply:
or as a property of the LV2 target?

Reply:
Ok in the end I put it with the CXX STANDARD, let's see if the CI balks :D